src/config/2019.json

```
@@ -311,6 +311,17 @@
       "github": "bkardell",
       "twitter": "briankardell"
     },
+    "borisschapira": {
+      "name": "Boris Schapira",
+      "bio": "Boris is Customer Success Manager at <a href=\"https://dareboost.com/en/\">Dareboost</a>. He's a WebPerf & Web Quality aficionado, a web professional, a developer, a teacher and loves standars, microtypography and chocolatines.",
+      "teams": [
+        "translators"
```

Go on. You've done some development in this and might need more from you before this translation is done 😀

```suggestion
        "developers",
        "translators"
```